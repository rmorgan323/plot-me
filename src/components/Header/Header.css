/******  GLOBAL CSS VARIABLES  ******/
/***  COLORS  ***/
/***  FONTS  ***/
* {
  font-size: 16px;
  font-family: "Open Sans", sans-serif;
  outline: none;
  color: #414042; }

h1, h2, h3, h4, h5, h6 {
  font-family: "Open Sans", sans-serif;
  line-height: 1.3; }

h1 {
  font-size: 44px; }

h2 {
  font-size: 30px; }

h3 {
  font-size: 24px; }

h4 {
  font-size: 18px; }

h5 {
  font-size: 14px; }

h6 {
  font-size: 10px; }

p {
  font-size: 14px;
  line-height: 1.3;
  font-weight: 900; }

::placeholder {
  opacity: .5;
  text-align: center; }

/***  MIXINS  ***/
.Header {
  background-color: #414042;
  color: #fff;
  padding: 0 30px 8px; }
  .Header h1 {
    font-family: "Open Sans", sans-serif;
    font-size: 30px;
    font-family: 'Refreshment Stand';
    font-size: 60px;
    margin: 0;
    color: #fff;
    font-weight: 400;
    text-align: left; }
  .Header .header-span {
    font-family: "Open Sans", sans-serif;
    font-size: 30px;
    font-size: 60px;
    font-family: 'Refreshment Stand';
    color: #fff;
    margin: 0; }
