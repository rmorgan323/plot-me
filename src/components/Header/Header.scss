@import '../../variables.scss';

.Header {
  background-color: $grey6;
  color: #fff;

  h1 {
    @include match-h2;
    margin: 0;
    color: #fff;
    font-weight: 400;
  }
}