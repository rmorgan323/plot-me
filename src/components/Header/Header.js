import React from 'react';
import './Header.css';
import { NavLink } from 'react-router-dom';

const Header = () => {

  return (
    <div className="Header">
      <NavLink to={'/plotme/index'}>
        <h1>plot<span className="header-span">me</span></h1>
      </NavLink>
    </div>
  )
}

export default Header;