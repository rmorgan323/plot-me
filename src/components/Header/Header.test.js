import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header tests', () => {

  it('matches the snapshot', () => {
    const renderedHeader = shallow(<Header />);

    expect(renderedHeader).toMatchSnapshot();
  });
});