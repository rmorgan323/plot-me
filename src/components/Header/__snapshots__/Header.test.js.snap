// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <NavLink
        activeClassName="active"
        ariaCurrent="true"
        to="/plotme/index"
>
        <h1>
                plot
                <span
                        className="header-span"
                >
                        me
                </span>
        </h1>
</NavLink>,
      "className": "Header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "activeClassName": "active",
        "ariaCurrent": "true",
        "children": <h1>
          plot
          <span
                    className="header-span"
          >
                    me
          </span>
</h1>,
        "to": "/plotme/index",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            "plot",
            <span
              className="header-span"
>
              me
</span>,
          ],
        },
        "ref": null,
        "rendered": Array [
          "plot",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "me",
              "className": "header-span",
            },
            "ref": null,
            "rendered": "me",
            "type": "span",
          },
        ],
        "type": "h1",
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <NavLink
          activeClassName="active"
          ariaCurrent="true"
          to="/plotme/index"
>
          <h1>
                    plot
                    <span
                              className="header-span"
                    >
                              me
                    </span>
          </h1>
</NavLink>,
        "className": "Header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "ariaCurrent": "true",
          "children": <h1>
            plot
            <span
                        className="header-span"
            >
                        me
            </span>
</h1>,
          "to": "/plotme/index",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "plot",
              <span
                className="header-span"
>
                me
</span>,
            ],
          },
          "ref": null,
          "rendered": Array [
            "plot",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "me",
                "className": "header-span",
              },
              "ref": null,
              "rendered": "me",
              "type": "span",
            },
          ],
          "type": "h1",
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
